### PR TITLE
Wishlist

### PR DESCRIPTION
**Pull request by poojakelkar**
wishlist

### CODE REVIEW

**Review by chanchal-panpaliya — APPROVED** [review]
code looks clean with proper naming with good UI

File: src/pages/Wishlist/index.jsx [hunk 1]
```diff
@@ -1,5 +1,94 @@
+import { DeleteOutlined } from "@material-ui/icons";
+import React from "react";
+import { Link } from "react-router-dom";
+import { useCounter } from "../../counterContext";
+import { Announce } from "../../womenFrontPage/Announce/index";
+import { Footer } from "../../womenFrontPage/Footer/index";
+import { LastFooter } from "../../womenFrontPage/LastFooter/index";
+import { Nav } from "../../womenFrontPage/Nav/index";
+import {
+  Bottom,
```
Comment (ivinayakg) [inline src/pages/Wishlist/index.jsx:10]: if there are components you are not rendering then remove them from here

File: src/pages/Wishlist/index.jsx [hunk 2]
```diff
@@ -1,5 +1,94 @@
+import { DeleteOutlined } from "@material-ui/icons";
+import React from "react";
```
Comment (ivinayakg) [inline src/pages/Wishlist/index.jsx:2]: no need to  import react any more after v16

**Review by ivinayakg — APPROVED** [review]
I like this approach of using styled components, probably the approach I'm going to follow from my next project onwards.
Great Work

File: src/pages/Wishlist/index.jsx (hunk 1 above)
Comment (poojakelkar) [reply src/pages/Wishlist/index.jsx:10]: yeah will do that, Thanks for review

**Pull request merged**